.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup JDK
         uses: actions/setup-java@v3
         with:
-          java-version: 11
```

Comment:
Does this mean the minimum JDK for builds is now 17? If so, you might want to mention it in the README (at the bottom). Also, I assume this does not mean it will drop the minimum JDK 8 support?

Reply:
Yeah, the minimum JDK for builds is 17 now

Reply:
@steve-todorov any comments on this?